Sources/StreamChatUI/CommonViews/ContainerView.swift

```diff
@@ -32,10 +32,16 @@ extension CGFloat {
 }
 
 public class ContainerView: UIView {
-    convenience init(axis: NSLayoutConstraint.Axis = .horizontal, alignment: Alignment = .fill, views: [UIView]) {
+    convenience init(
+        axis: NSLayoutConstraint.Axis = .horizontal,
+        alignment: Alignment = .fill,
+        views: [UIView] = [],
+        spacing: CGFloat = .auto
```

Comment:
can we move `spacing` before `views`? It's nice to have the array as the last parameter.